src/tools/clippy/tests/ui/iter_overeager_cloned.stderr

```diff
@@ -28,6 +28,17 @@ LL |     let _: usize = vec.iter().filter(|x| x == &"2").cloned().count();
    = note: `-D clippy::redundant-clone` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::redundant_clone)]`
 
+error: taken reference of right operand
```

Comment:
This lint shouldn't trigger in the test file. Please fix the two new occurrences  by removing the ref.

Reply:
Alright, will fix.